src/Microsoft.DotNet.Build.Tasks/PackageFiles/GenerateAssemblyInfo.Sdk.targets

```diff
@@ -0,0 +1,66 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <PropertyGroup>
+    <!-- BuildTools never generated this attribute before, so don't let the SDK generate it. -->
+    <GenerateAssemblyConfigurationAttribute>false</GenerateAssemblyConfigurationAttribute>
+
+    <!--
+    When using the Microsoft.NET.Sdk, Company and Product are defaulted to $(AssemblyName)
+    in Microsoft.NET.Sdk.BeforeCommon.targets, which means we would have to default them in
+    a props file. However, the versioning.props file has to be imported during versioning.targets
+    in order to get all the version properties set correctly.
+    So overwrite these two properties here, so they are set correctly.
+    -->
+    <Company>Microsoft Corporation</Company>
+    <Product>Microsoft%AE .NET Framework</Product>
+  </PropertyGroup>
+
+  <Target Name="_ComputeBuildToolsAssemblyInfoAttributes"
+          DependsOnTargets="CreateVersionFileDuringBuild;DecideIfWeNeedToIncludeDllSafeSearchPathAttribute"
+          BeforeTargets="GetAssemblyAttributes">
+    <ItemGroup>
+      <AssemblyAttribute Include="System.Reflection.AssemblyDefaultAliasAttribute">
```

Comment:
Is it worth sharing more logic between the sdk and nonsdk version of these files and essentially just condition an extra set of item groups?

Reply:
My hope was the nonsdk version just goes away and we pull the SDK version back into versioning.targets.

I'm not seeing how more logic can be shared.  Can you explain more?

Reply:
The writing of the assmeblyinfolines into a file is all shared. In once case you simply need to add more items to the itemgroup then the other case. At any rate I agree with you that I only want the one at the end of the day so I'm not that worried about it.